internal/registry/labels.go

```diff
@@ -34,22 +34,22 @@ func (e MetadataNotFoundError) Error() string {
 }
 
 // Labels is a set of key:value labels from an operator-registry object.
-type Labels map[string]string
+type LabelsMap map[string]string
```

Comment:
Sure.

Reply:
Updated.

`This changes raised one issue where Labels field is used by ginkgo/v2. This field is already defined in SDK which was causing the issue. I updated the SDK field Labels to LabelsMap to avoid the issue causing due to same name.
`